geomesa-quickstart-kafka/README.md

```diff
@@ -63,7 +63,7 @@ Run the Code
 On the command-line, run:
 
 ```bash
-$ java -cp geomesa-quickstart-kafka/target/geomesa-quickstart-kafka-${geomesa.version}.jar com.example.geomesa.kafka.KafkaQuickStart -brokers <brokers> -zookeepers <zookeepers>
+$ java -cp geomesa-quickstart-kafka/target/geomesa-quickstart-kafka-$VERSION.jar com.example.geomesa.kafka.KafkaQuickStart -brokers <brokers> -zookeepers <zookeepers>
```

Comment:
Is there any sensible way for us to handle variable filtering in these README.md's?